cpp/test/Slice/errorDetection/WarningInvalidComment.err

```diff
@@ -0,0 +1,3 @@
+WarningInvalidComment.ice:10: warning: ignoring unknown doc tag '@remarks' in comment
```

Comment:
Should we refer to these comments as "doc comment[s]"?

I find the diagnostic "ignored xxx in comment" odd as everything in plain comment should be ignored.
If I write:
```
// @foo in a plain comment
```

I don't anticipate any diagnostic.